migrations/sqlite/20240809205107_event_source_anchored.up.sql

```diff
@@ -0,0 +1,9 @@
+-- Add up migration script here
```

Comment:
looks like the file names for these migrations probably should be updated in a follow-up since they don't really reflect what is happening in them anymore

Reply:
e.g. they don't add "anchored" anymore

Reply:
Yes, we fixed this in part 5.

Reply:
Oh dang, I didn't notice that. Unfortunately, I don't think you can change a migration name after it's been applied somewhere.. pretty sure it's a unique constraint or part of the hash with how sqlx manages them. We can try it but iirc it will error saying something like migration with name x has previously been applied but is missing.

Reply:
presumably so long as we fix this before it goes out in a release it's fine?

Reply:
well, it's in a prerelease now and if anyone has run main locally they'll need to manually adjust their db (or delete the file). plus any test nodes we're running it on... my opinion is that's it a bummer but isn't worth the risk/hassle. I don't really pay attention to migration names though (clearly) and just look at the schemas.